bundles/org.openhab.binding.tuya/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -185,6 +185,33 @@
 		</config-description>
 	</channel-type>
 
+	<channel-type id="quantity">
+		<item-type>Number</item-type>
+		<label>Number</label>
+
+		<category>Number</category>
+
+		<state pattern="%.3f"></state>
+
+		<config-description>
```

Comment:
This configuration should be documented

Reply:
Where and how? Quantity channel types only have configuration because the others do and the handler code depends on it. Unlike the other channel types quantity channels aren't user-addable and depend on the device schema. (Probably the others shouldn't be addable at this stage - it just let's end users generate unsupportable snowflakes - but that's not relevant to this PR.)

Reply:
I noted it in the README for now.